templates/zerver/help/view-and-edit-your-message-drafts.md

```diff
@@ -0,0 +1,72 @@
+# View and edit your message drafts
+
+Message drafts are created when unsent messages are discarded by exiting out
+of the compose box while content has been written in the **Compose your message here...**
+field. Zulip's [Drafts](/#drafts) feature allows users to retrieve and edit
+draft contents in order to send them as new messages.
+
+## View your drafts
+
+To view a list of all your currently saved drafts, you can open the
+[Drafts](/#drafts) modal by clicking the **Drafts** button on the left side of
+the compose box located at the bottom of your window or pressing the `d` key.
```

Comment:
You might want to also talk about the blue Drafts button when your compose box is open.